bin/node/src/chain_spec.rs

```diff
@@ -141,6 +144,11 @@ impl ChainParams {
             .unwrap_or(DEFAULT_MILLISECS_PER_BLOCK)
     }
 
+    pub fn base_unit_creation_delay(&self) -> u64 {
```

Comment:
Can we return BaseUnitCreationDelay here? And to make the code consistent, can you change the return types of `ChainParams::millisecs_per_block()` and `ChainParams::session_period()` to `MillisecsPerBlock` and `SessionPeriod`, respectively? By the way, shouldn't these types should probably defined in `aleph-primitives`?